.github/workflows/release-new-app.yml

```diff
@@ -0,0 +1,37 @@
+name: New release app
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+  packages: read
```

Comment:
Sorry, one more thing. I don't think these permissions are necessary since this action doesn't install any packages etc.